qiskit/pulse/pulse_lib/samplers/decorators.py

```diff
@@ -135,9 +135,10 @@ def linear(times: np.ndarray, m: float, b: float) -> np.ndarray:
 
 import numpy as np
 
-from qiskit.pulse.samplers import strategies
 import qiskit.pulse.commands as commands
 
+from . import strategies
```

Comment:
can we use `from qiskit.pulse.pulse_lib.import` 😍 ? It's okay if not

Reply:
The Qiskit convention thus far seems to be to use relative imports, but it does not seem there is standardization throughout. Relative imports are not explicitly discouraged anymore in PEP 328. If they had been used originally in this module the line wouldn't have had to change when moving the module. Thoughts @ajavadia, @kdk?

Reply:
I should add, I'm ok either way.

Reply:
Ik qiskit usually uses relative imports; I guess I got excited that there was a file that had absolute imports 😁 I'm also alright either way